packages/lib/server-only/document/seal-document.ts

```diff
@@ -45,7 +45,13 @@ export const sealDocument = async ({ documentId, sendEmail = true }: SealDocumen
     },
   });
 
-  if (recipients.some((recipient) => recipient.signingStatus !== SigningStatus.SIGNED)) {
+  if (
+    recipients.some(
+      (recipient) =>
+        recipient.signingStatus !== SigningStatus.SIGNED &&
+        (recipient.role === RecipientRole.SIGNER || recipient.role === RecipientRole.APPROVER),
+    )
+  ) {
```

Comment:
The alternative is we make something like this an extracted method and use it in lieu of the the big query above and then reuse it in places like here.

Reply:
The approach of CC having SIGNED by default was nice!